tools/sv-report

```diff
@@ -249,3 +263,41 @@ except KeyError:
     logger.critical("Unable to generate report, not enough logs")
 except Exception as e:
     logger.critical("Unable to generate report: " + str(e))
+
+csv_header = ['name', 'files', 'tags']
+csv_output = {}
+duplicates = []
+
+for r in data:
+    csv_header.append(r)
+    for test in data[r]["tests"]:
+        test_handle = data[r]["tests"][test]
+        name = test_handle["name"]
+        files = getRelativePaths(test_handle["files"])
+
+        try:
+            if csv_output[name]["files"] != files:
+                logger.error("Duplicate test: {}, first use: {}, second: {}"
+                             .format(name, csv_output[name]["files"],
+                                     files))
+                if name not in duplicates:
+                    duplicates.append(name)
+            else:
+                csv_output[name][r] = test_handle["status"] == "test-passed"
+        except KeyError:
+            csv_output[name] = {}
+            csv_output[name]["name"] = test_handle["name"]
+            csv_output[name]["files"] = files
```

Comment:
I've noticed that there is a space after the file field in the generated csv file, is that desired?

an example:
```
repeat_loop,tests/chapter-12/12.7.2--repeat.sv ,12.7.2,False,True
jump_return_expr,tests/chapter-12/12.8--return_val.sv ,12.8,False,True
for_loop,tests/chapter-12/12.7.1--for.sv ,12.7.1,False,True
if,tests/chapter-12/12.4--if.sv ,12.4,True,True
```

Reply:
This is now fixed